x-pack/plugins/siem/public/common/components/exceptions/helpers.tsx

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiText, EuiCommentProps, EuiAvatar } from '@elastic/eui';
+import { flatten, capitalize } from 'lodash';
+import moment from 'moment';
+
+import * as i18n from './translations';
+import {
+  FormattedEntry,
+  OperatorType,
+  OperatorOption,
+  ExceptionEntry,
+  NestedExceptionEntry,
+  DescriptionListItem,
+  Comment,
+  ExceptionListItemSchema,
+} from './types';
+import { EXCEPTION_OPERATORS, isOperator } from './operators';
+
+export const getEmptyValue = () => '—';
```

Comment:
You don't want to copy this function here, but rather just use the existing one from:

```ts
x-pack/plugins/siem/public/common/components/empty_value/index.tsx
```

Reply:
Updated!